components/Tabs/test.js

```diff
@@ -0,0 +1,81 @@
+
+class TabLink {
+    constructor(element) {
+      // Assign this.element to the passed in DOM element
+      // this.element;
+        this.element = element
+      // Get the custom data attribute on the Link
+      // this.data;
+        this.data = this.element.dataset.tab
+      // Using the custom data attribute get the associated Item element
+      // this.itemElement;
+        this.itemElement = document.querySelector(`div[data-tab='${this.data}'].tabs-item`)
+        console.log(this.itemElement)
```

Comment:
Good job on using console.logs to check and debug your code, but don't forget to remove them when you are done.